astropy/modeling/core.py

```diff
@@ -36,52 +34,61 @@
 from ..units import Quantity, UnitsError, dimensionless_unscaled
 from ..units.utils import quantity_asanyarray
 from ..utils import (sharedmethod, find_current_module,
-                     InheritDocstrings, OrderedDescriptorContainer,
+                     InheritDocstrings,
                      check_broadcast, IncompatibleShapeError, isiterable)
+from ..utils.misc import get_parameters
 from ..utils.codegen import make_function_with_signature
 from ..utils.exceptions import AstropyDeprecationWarning
-from .utils import (combine_labels, make_binary_operator_eval,
+from .utils import (make_binary_operator_eval,
                     ExpressionTree, AliasDict, get_inputs_and_params,
                     _BoundingBox, _combine_equivalency_dict)
 from ..nddata.utils import add_array, extract_array
+#from . import compound
+from .parameters import (Parameter, InputParameterError,
+                         param_repr_oneline, _tofloat)
 
-from .parameters import Parameter, InputParameterError, param_repr_oneline
-
-
-__all__ = ['Model', 'FittableModel', 'Fittable1DModel', 'Fittable2DModel',
-           'custom_model', 'ModelDefinitionError']
+from collections import deque
 
```

Comment:
Removing the `__all__` here is a pretty significant API change. Do we intended for none of these to be importable directly through `astropy.modeling` anymore? If so, then we require a deprecation period here. If not, then this should be restored and `CompoundModel` should be added.

Reply:
looks like a mistake, added back in.